vimeo-networking/build.gradle

```diff
@@ -67,19 +67,50 @@ artifacts {
     archives sourcesJar, javadocJar
 }
 
+// Create the pom configuration:
+def pomConfig = {
+    licenses {
+        license {
+            name "MIT License"
+            url "http://www.opensource.org/licenses/mit-license.php"
+            distribution "repo"
+        }
+    }
+    developers {
+        developer {
+            id "vimeo"
+            name "Vimeo Mobile"
+            email "mobileops@vimeo.com"
+            organisation "Vimeo"
+            organisationUrl "https://vimeo.com"
```

Comment:
maybe change this to the github url? im guessing this shows up in the bintray package list?